### PR TITLE
Creating the layout with heade and footer

### LINKED ISSUE #17: Add SummaryForm Component

### PR DESCRIPTION
**Pull request by govindarajkp**
Fixes #17

### **Summary**
1. Creating layout module for the application, which will contains the region module to integrate the headers and footer components across the application.
2.  Adding the new  router as summary to show the summary form. 

**### Steps to do this Tech Update**
Angular CLI -
ng g c layout/simple
ng g m layout/simple
ng generate c layout/region
ng generate m layout/region

ng generate c layout/region/header
ng generate c layout/region/footer
ng generate c layout/region/left-side-bar
ng generate c summary-form

Creates - 
src/app/layout/simple
    src/app/layout/region
        src/app/layout/header
       src/app/layout/footer
       src/app/layout/left-side-bar
src/app/summary-form

## Checklist

- [x] Have you added the summary of what your changes do and why you'd like us to include them?
- [x] Have you added the steps to implement this feature?
- [ ] Have the files been linted and formatted?
- [ ] Have the docs been updated to match the changes in the PR?
- [ ] Have the tests been updated to match the changes in the PR?
- [ ] Have you run the tests locally to confirm they pass?

### How does this PR fix the problem, Screenshot Please?

### CODE REVIEW

File: src/app/layout/region/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<p>footer works!</p>
```
Comment (ramsunvtech) [inline src/app/layout/region/footer/footer.component.html:1]: @govindarajkp PR Changes should be like below. so its multiple PRs.

1. Simple Layout Component
2. Simple Layout Header Component
3. Simple Layout MainMenu Component
4. Simple Layout SideBar Component
5. Simple Layout Footer Component
6. Summary Form

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

**Review by shangan23 — CHANGES_REQUESTED** [review]
@govindarajkp 
- <s>Header, Footer, and NotFound component please move it under `Shared module`. </s>
- <s>Replace shared component in `app.module.ts` with `SharedModule`</s>
- From next PR keep the changeset simple 29 file changes are hard to follow for this kind of changeset raise multiple PR's 
- Commit message has to be in standard `Creating the layout with heade and footer`  with two commits 
     1.  `tech(component/header): add header component`
     2. `tech(component/footer): add header footer`
- Issue ID #17 includes only `SummaryForm component` but this PR contains many things including Header, Footer, Simple Layout ... etc

File: src/app/layout/simple/simple.module.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { NgModule,NO_ERRORS_SCHEMA } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import { SimpleComponent } from './simple.component';
```
[the remark this answers is not in the document]
Comment (govindarajkp) [reply src/app/layout/simple/simple.module.ts:4]: On the layout we have a module name  as simple, still need name as simpleLayout ?

**Comment by shangan23** [comment]
@JP - that was a wrong merge.

The PR had label's DO NOT MERGE
<https://github.com/bytebraintechnologies/hrms-ui/labels/DO%20NOT%20MERGE>REVIEW
IN PROGRESS
<https://github.com/bytebraintechnologies/hrms-ui/labels/REVIEW%20IN%20PROGRESS>

It's in progress
